examples/webserver/fault-tolerance/src/main/java/io/helidon/webserver/examples/faulttolerance/FtService.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.webserver.examples.faulttolerance;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import io.helidon.common.reactive.Single;
+import io.helidon.faulttolerance.Async;
+import io.helidon.faulttolerance.Bulkhead;
+import io.helidon.faulttolerance.CircuitBreaker;
+import io.helidon.faulttolerance.Fallback;
+import io.helidon.faulttolerance.Retry;
+import io.helidon.faulttolerance.Timeout;
+import io.helidon.webserver.Routing;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.Service;
+
+/**
+ * Simple service to demonstrate fault tolerance.
+ */
+public class FtService implements Service {
```

Comment:
May be a bit more readable if methods had a suffix like "M", a bit difficult to read when named same as variables

Reply:
Will make it clear

Reply:
Changed method names to `*Handler`